src/apps/backend/modules/communication/sms-service.ts

```diff
@@ -7,7 +8,8 @@ export default class SMSService {
   public static async sendSMS(params: SendSMSParams): Promise<void> {
     const enableSMS = ConfigService.getValue('enableSMS');
 
-    if (!enableSMS) {
+    // If SMS is disabled or the recipient phone number is the default phone number, do not send SMS
+    if (!enableSMS || isDefaultPhoneNumber(params.recipientPhone.phoneNumber)) {
```

Comment:
@abhishek310001 - Let's keep this implementation as it was before. The _sms_ can be anything here and it's not tied to OTP. We can have this check on otp/user level instead.

Reply:
Also let's add this - If sms is not enabled, show warning using logger - `SMS not enabled. Could not send message - ${params.messageBody}`

Reply:
Added logger warning and reverted to default conditions.